setup.md

```diff
@@ -213,7 +215,7 @@ $ git config --global core.editor "vim"
 ~~~
 {: .bash}
 
-A more complete list of possible editors is available [here](http://swcarpentry.github.io/git-novice/02-setup/index.html).
+A more complete list of possible editors is available: [Setting Up Git](http://swcarpentry.github.io/git-novice/02-setup/index.html).
```

Comment:
I am fine removing the word [here] from links. However, we could instead cite the source for this and edit the text to be more descriptive.

A more complete list of possible editors is available [from Software Carpentry](http://swcarpentry.github.io/git-novice/02-setup/index.html)